src/Modules/CalcOffence.lua

```diff
@@ -3858,20 +3852,29 @@ function calcs.offence(env, actor, activeSkill)
 				end
 			end
 
-			local bleedRollAverage = bleedStacks / (bleedStacks + 1) * 100
+			-- the amount of damage each bleed does as % maximum
```

Comment:
This change should also be done on ignites which uses the same `local igniteRollAverage = igniteStacks / (igniteStacks + 1) * 100`

Reply:
Integrated similar changes to ignite and poison.